### PR TITLE
Makefile for issue #2

### PR DESCRIPTION
**Pull request by Flukas88**
Added a minimal Makefile even though Cmake would be a better tool

### CODE REVIEW

File: src/Makefile [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.DEFAULT_GOAL := release
+
+### Vars
+APPLICATION := uwf
```
Comment (dreamos82) [inline src/Makefile]: uwf??? :dagger:  :)

**Comment by dreamos82** [comment]
If i do just a make, it creates a relase-xyz empty folder, the compiled file stays in the same folder as the source. 
Also (this is up to you more a general question) shouldn't be better to put the makefile in the main folder instead of src?

File: src/Makefile (hunk 1 above)
Comment (Flukas88) [reply src/Makefile]: I plead the fifth

**Comment by Flukas88** [comment]
> If i do just a make, it creates a relase-xyz empty folder, the compiled file stays in the same folder as the source.
> Also (this is up to you more a general question) shouldn't be better to put the makefile in the main folder instead of src?

I forgot to put the folder prefix in the -o :)

As for the root folders, generally, you put it where your main() resides

**Comment by dreamos82** [comment]
Looks good to me! :+1:  thanks!

**Pull request merged**